src/status_im/contexts/wallet/events.cljs

```diff
@@ -355,19 +357,61 @@
 (rf/reg-event-fx :wallet/get-keypairs get-keypairs)
 
 (rf/reg-event-fx :wallet/bridge-select-token
- (fn [{:keys [db]} [{:keys [token token-symbol stack-id start-flow?]}]]
-   (let [missing-recipient? (-> db :wallet :ui :send :to-address nil?)
-         to-address         (-> db :wallet :current-viewing-account-address)]
+ (fn [{:keys [db]}
+      [{:keys [token token-symbol stack-id network owners network-details start-flow?] :as params}]]
+   (let [{:keys [wallet]}       db
+         unique-owner           (when (= (count owners) 1)
+                                  (first owners))
+         token                  (if (and unique-owner (nil? token))
+                                  (let [token (utils/get-token-from-account db
+                                                                            token-symbol
+                                                                            unique-owner)]
+                                    (assoc token
```

Comment:
There are few places in our codebase where `token` structure extended by the same set of fields. Sometime in the future it is probably worth asking question "why this happens")
But in the meanwhile it would be nice to make such extension at least consistent.

I see there is a `utils/tokens-with-balance` function that does same transformation on a bunch of tokens. I'd suggest to dedicate a separate utility function to work on one token and reuse it here, in `utils/tokens-with-balance` and all other places where same extension happens (for example in `:wallet/select-from-account`).

Wdyt?

Reply:
Sounds good, just added `utils/token-with-balance` function and used it here, in `:wallet/select-from-account` event handler and in `tokens-with-balance` function